sqllogictest-bin/src/lib.rs

```diff
@@ -170,8 +170,16 @@ pub async fn main_okk() -> Result<()> {
         Color::Auto => {}
     }
 
-    let files = glob::glob(&files).context("failed to read glob pattern")?;
-    let files = files.into_iter().try_collect::<_, Vec<_>, _>()?;
+    let files: Vec<PathBuf> = files
+        .iter()
+        .map(|filename| {
+            glob::glob(filename)
+                .context("failed to read glob pattern")
+                .unwrap()
```

Comment:
Can we return error instead of unwrap here?

Reply:
... and also the `unwrap` below

Reply:
Fixed, thank you for your advice!